gateway-provider-ha/src/main/java/org/apache/knox/gateway/ha/provider/impl/HaServiceConfigConstants.java

```diff
@@ -53,4 +59,6 @@
    boolean DEFAULT_NO_FALLBACK_ENABLED = false;
 
    String DEFAULT_STICKY_SESSION_COOKIE_NAME = "KNOX_BACKEND";
+
+   String DEFAULT_DISABLE_LB_USER_AGENTS = "ClouderaODBCDriverforApacheHive";
```

Comment:
What if the default includes multiple values in the future? Should this be an array type instead?

Reply:
Since this is coming from the HA config and will be a string I had it set up to mimic the config but looking at Sandor's and your comment looks like array/list might be appropriate.